bin/gce/install.sh

```diff
@@ -55,12 +55,16 @@ if [[ "${ROLE}" == 'Master' ]]; then
        stable"
```

Comment:
Can you add a comment at the top of this script summarizing everything this script is installing?